fms_core/viewsets.py

```diff
@@ -389,6 +397,12 @@ def get_serializer_class(self):
             return NestedSampleSerializer
         return SampleSerializer
 
+    @action(detail=False, methods=["get"])
+    def list_export(self, _request):
+        samples = Sample.objects.all().prefetch_related("container", "individual")
```

Comment:
I think you should use select_related here since those are 1 to 1 relations. This should be more efficient.